source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
-* After the item has been requested the user should then be denied access. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Title Report audit tests for books and reference works apply to Host_Types Aggregated_Full_Content, eBook, and eBook_Collection.
 
-Each of these concurrent/simultaneous turnaways must report 1 Limit_Exceeded in the DR_D2 Standard View.
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
 
+E.2.4.1 Unique_Title_Investigations and Unique_Title_Requests
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Audit Test D2-2: No_License
-...........................
+**Option 1**: Book_Segments and/or Reference_Items are available, users can elect to access Books and/or Reference_Works segment-by-segment.
 
-**IMPORTANT NOTE**: This test can only be carried out if the content provider restricts site content or if restricted content is not displayed. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
+The auditor MUST request 100 Book_Segments and/or Reference_Items, 10 each from 10 different Books and/or Reference_Works. This MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 10 each of Unique_Title_Investigations and Unique_Title_Requests.
 
-The account used for this testing must have restricted access to content. The content the user has no license to access must be declared by the content provider prior to the testing. The auditor will attempt to access content using the account set up with restricted access. Each time access is refused, the auditor will record 1 No_License.
+**Option 2**: Only whole Books and/or Reference_Works are available.
 
-Each of these No_License turnaways must report 1 No_License in the DR_D2 Standard View.
+The auditor MUST request 20 Books and/or Reference_Works, twice each. This MUST result in 40 each of Total_Item_Investigations and Total_Item_Requests, and 20 each of Unique_Item_Investigations, Unique_Item_Requests, Unique_Title_Investigations and Unique_Title_Requests.
 
+**Option 3**: Books and/or Reference_Works are available as both whole Books and/or Reference_Works and as Book_Segments and/or Reference_Items.
 
-E.2.3 Title Reports
-"""""""""""""""""""
+The auditor MUST request 20 Books and/or Reference_Works, twice each, recording the number of Book_Segments and/or Reference_Items pertaining to each title.
 
-E.2.3.1 Master Report: TR
-'''''''''''''''''''''''''
+This MUST result in 20 each of Unique_Title_Investigations and Unique_Title_Requests, and a count of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests that is equal to the sum of the Book_Segments and/or Reference_Items.
```

Comment:
This is correct for the Unique_Item metrics, but the Total_Item metrics must be twice the Unique_Item metrics.